arduino/smartcar/smartcar.ino

```diff
@@ -49,7 +58,12 @@ void setup()
     Serial.begin(9600);
 }
 
+// Distance of our choice is 25cm from the wall.
 void loop()
 {
-  
-}
+    car.setSpeed(70);
+    while(!(frontUS.getDistance() == 0) && frontUS.getDistance() < 90 ){
```

Comment:
There are two issues:
1. Why a loop (i.e. `while`) inside another loop (i.e. `loop()`)? If you want something to only happen once like setting the speed, you place it in `setup().
2. You are calling `getDistance` **twice** which means you will get two **different** measurements. Is this really what you want? What happens if the first time the sensor returns 100 and the next time it returns 0?

Reply:
I believe my colleague has pushed the assignment with the necessary changes!.